tests/unit/example.spec.js

```diff
@@ -1,12 +1,33 @@
-import { shallowMount } from '@vue/test-utils'
-import HelloWorld from '@/components/HelloWorld.vue'
+// import { shallowMount } from '@vue/test-utils'
+// import HelloWorld from '@/components/HelloWorld.vue'
+//
+// describe('HelloWorld.vue', () => {
+//   it('renders props.msg when passed', () => {
+//     const msg = 'new message'
+//     const wrapper = shallowMount(HelloWorld, {
+//       propsData: { msg }
+//     })
+//     expect(wrapper.text()).toMatch(msg)
+//   })
```

Comment:
why do we have commented out code here?

Reply:
this one is an example script for review and future implementation